diktat-rules/src/main/resources/diktat-analysis.yml

```diff
@@ -455,6 +455,8 @@
 # Check that lambda with inner lambda doesn't use implicit parameter
 - name: PARAMETER_NAME_IN_OUTER_LAMBDA
   enabled: true
+  configuration:
+    strictMode: true # don't let outer lambdas have `it` as parameter
```

Comment:
Add configuration to the other `diktat-analysis.yml` files as well

Reply:
Done